backend/src/@types/models/Session.d.ts

```diff
@@ -3,14 +3,12 @@ type SessionId = number;
 interface SessionAttributes {
 	id: SessionId;
 	name: string;
-	userId: UserId;
```

Comment:
Is this a good idea ?

Reply:
The same reason we deleted userId in folder. Its original purpose is for access permission, but it has courseId already.

Reply:
I thought course id was the long term course id so that way the questions are connect to a course over many semester 

if that's the case the creator of the session and folder cant not be determined since the professor can change from semester to semester

is this the way we want things to work?

Reply:
if so then just fix the type issue LJ mentioned and this will be good to go

Reply:
No, courseId is only 1 semester.

Reply:
I think that only the creator should have access to the content. That is why I suggested creating a course table. But you were against it.

Reply:
Does course table have a user id 

If yes then ignore my comment and fix LJs thing

Reply:
The course table should only has courseId and userId